packages/plugin-infra/src/hooks/usePluginWrapper.tsx

```diff
@@ -24,7 +24,7 @@ export function usePluginWrapper(open: boolean, options?: { width?: number; name
 
     useEffect(() => {
         setWrap(open)
-        return () => setWrap(false)
+        return () => setWrap(open)
```

Comment:
hmm wait, this is strange...

Reply:
Why does it let the child component decide how the parent component should behave?

Reply:
because it behave like a context 👀

Reply:
@yanzhihong23 Is this operation useless?